009-migrate-quasar-runtime.md

```diff
@@ -0,0 +1,127 @@
+
+
+# Migrate Quasar's jobs runtime
+
+We want to simplify our automated job's runtime by migrating from Jenkins (EC2) to a simpler alternative (Hosted).
+
+**Discussion:** <https://github.com/DoSomething/rfcs/pull/9>
+
+## Problem
+
+The most challenging task for data engineers is to build the adequate plumbing necessary in order to provide analysts with clean and timely data. We identify the sources of the data, extract it, and transport it to our warehouses. Once there, we transform it from it's raw form (dirty) into analysis ready data (clean). This process is repeated per data source, is known as ETL orchestration.
+
+A central part of this orchestration is to schedule and run jobs in an automated fashion. We use Jenkins for this. Although we have adopted better processes and tools like Fivetran (E), DBT (T), and Looker (L). We have identified Jenkins as a source of frustration for the team. Both seasoned engineers and new members find troubleshooting and maintaining both instances (QA, Prod) particularly time consuming. It also adds a significant slope to the learning curve for new members because env variables, runtime dependencies, and logs are inside these boxes. Basically, new members will need to pull their DevOps/SisAdmin hats just to be able to setup their local environment correctly, before troubleshooting any data pipeline issues.
```

Comment:
@rapala61 I think the breakdown of our tools by ETL is closer to:
Fivetran (EL)
DBT (T)

Looker does some T, depending on user generated or persisted dashboards, but I don't think that's what we're trying to capture in this RFC.